sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/exception/ErrorCode.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.stargate.sgv2.docsapi.api.exception;
+
+import io.stargate.sgv2.docsapi.api.common.exception.model.dto.ApiError;
+import io.stargate.sgv2.docsapi.config.DocsApiConfiguration;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.jboss.resteasy.reactive.RestResponse;
+import org.jboss.resteasy.reactive.RestResponse.ResponseBuilder;
+
+/** Simple enumeration for the error code that can provide back the correct response to the user. */
+public enum ErrorCode {
+
+  /** Generic data store errors. */
+  DATASTORE_KEYSPACE_DOES_NOT_EXIST(Response.Status.NOT_FOUND, "An unknown namespace provided."),
+
+  DATASTORE_TABLE_DOES_NOT_EXIST(Response.Status.NOT_FOUND, "An unknown table provided."),
+
+  DATASTORE_TABLE_NAME_INVALID(
+      Response.Status.BAD_REQUEST,
+      "The table name contains invalid characters. Valid characters are alphanumeric and underscores."),
+
+  /** Document API. */
+  DOCS_API_GENERAL_ARRAY_LENGTH_EXCEEDED(
+      Response.Status.BAD_REQUEST,
+      String.format(
+          "Max array length of %s exceeded.", DocsApiConfiguration.DEFAULT.getMaxArrayLength())),
```

Comment:
I am not sure that we will have `DocsApiConfiguration` anymore.. Can you please avoid using it for now and copy the values directly in the default error messages.